docs/authentication/social-connections/account-linking.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Account Linking
+description: Learn how Clerk handles account linking and manages unverified email addresses from OAuth providers.
+---
+
+# Account Linking
+
+Account Linking is a process used by Clerk to ensure a smooth sign-in and sign-up experience using both OAuth and other methods (e.g. username/password). By using the "email address" as the common identifier, Clerk aims to merge accounts whenever possible. This mechanism is triggered when an OAuth provider returns an email address that matches an existing account, assuming a single owner for each email address.
+
+## How it works
+
+When a user attempts to sign-in or sign-up, Clerk validates the email address provided. If the email comes from an OAuth provider, there are several scenarios depending on the verification status of the email and whether an account with the same email already exists. Below are the different scenarios and how Clerk handles them.
+
+<Images
+    width={3333}
+    height={3551}
+    src="/docs/images/authentication/account-linking-flow.webp"
+    alt="Flow chart of the account linking process in various scenarios"
+/>
+
+### When email address is verified in both OAuth and Clerk
+
+In this scenario, when a user with a verified email address signs in using an OAuth provider that returns a matching verified email, Clerk links the OAuth account to the existing account and signs the user in. This is valid even if the existing account has a password.
```

Comment:
:upside_down_face: Let's expand on the "even if has a password" case? Explain that the user will be signed in skipping the password check.

Reply:
✅ Changed the sentence a bit.